ci/conda/recipes/morpheus-libs/meta.yaml

```diff
@@ -13,19 +13,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# This recipe splits into packages - morpheus-core, morpheus-dfp and morpheus-llm
 {% set version = environ.get('GIT_VERSION', '0.0.0.dev').lstrip('v') + environ.get('VERSION_SUFFIX', '') %}
 {% set minor_version =  version.split('.')[0] + '.' + version.split('.')[1] %}
 
 package:
-  name: morpheus-core
+  name: morpheus-split
   version: {{ version }}
 
 source:
   git_url: ../../../..
 
 outputs:
-
+  ############################### morpheus-core #############################
   - name: morpheus-core
+    type: conda_v2
```

Comment:
Using type v2 results in a `.conda` file vs the `.tar.bz2` file. `.conda` files are expected to have better download and extract performance - https://www.anaconda.com/blog/understanding-and-improving-condas-performance